src/main.rs

```diff
@@ -462,7 +523,9 @@ fn build_ui(app: &App, application: &gtk::Application) {
     // vertically or horizontally
     let vbox = gtk::Box::new(gtk::Orientation::Vertical, 0);
     vbox.pack_start(&view, true, true, 0);
-    window.add(&vbox);
+
+    overlay.add(&vbox);
+    window.add(&overlay);
```

Comment:
Maybe we could only create the overlay on demand when needed?

Reply:
Seems simpler this way. Like that it's initiated at the beginning once and for all, and we just change its visibility.

Reply:
OK :)